### DISCUSSION TITLE
Misorientation angle vs Length of Boundaries

### DISCUSSION THREAD

**Discussion by brittcolea** [mtex-toolbox/mtex#1415] [Ask Anything]
Hello,

I'm looking to plot the misorientation angle vs length of boundaries per unit area as shown here: 
![image](https://user-images.githubusercontent.com/42809545/165832705-9f8e2267-c6d2-4b60-9be7-e9d40e6e63ec.png)
https://mdpi-res.com/d_attachment/materials/materials-10-00730/article_deploy/materials-10-00730-v3.pdf?version=1499325749

I was attempting to use this tutorial: [https://mtex-toolbox.github.io/HomepageOld/files/doc/GrainMisorientationAnalysis.html](url) but I am not sure how to further manipulate the way these variables are saved. How would I do this?

Thanks

**Comment by kilir** [comment]
Hi,
misorientation angle over a boundary segment is stored in `grains.boundary.misorientation.angle` and the length of the segment in `grains.boundary.segLength`.

Cheers,
Rüdiger
